gatsby-config.js

```diff
@@ -20,10 +21,11 @@ const languages = [
 
 module.exports = {
   siteMetadata: {
-    title: `Chicago COVID Resources`,
+    siteTerritory: process.env.SITE_TERRITORY,
+    title: `${process.env.CITY} COVID Resources`,
     siteUrl,
-    author: `City Bureau`,
-    twitterAuthor: `@city_bureau`,
+    author: process.env.SITE_AUTHOR,
```

Comment:
owner/author are probably the same thing for practical purposes, but I wasn't sure if City Bureau would want to be listed as an author if something was deployed that was a derivative of the initial project? I think the idea is more to provide info on ownership and maintenance of the deployed site, though.